runtime/vm/j9vm.tdf

```diff
@@ -786,7 +786,7 @@ TraceEvent=Trc_VM_classInitStateMachine_verifyFlattenableField Group=classinit O
 TraceEvent=Trc_VM_classInitStateMachine_prepareFlattenableField Group=classinit Overhead=1 Level=3 Template="prepare flattenable field clazz=%p"
 TraceEvent=Trc_VM_classInitStateMachine_initFlattenableField Group=classinit Overhead=1 Level=3 Template="initialize flattenable field clazz=%p"
 
-TraceEntry=Trc_VM_GetCompleteNPEMessage_Entry Overhead=1 Level=3 Template="GetCompleteNPEMessage - bcCurrentPtr (0x%p) bcCurrent (0x%x) romClass (0x%p) npeCauseMsg (%s)"
+TraceEntry=Trc_VM_GetCompleteNPEMessage_Entry Overhead=1 Level=3 Template="GetCompleteNPEMessage - bcCurrentPtr (0x%p) romClass (0x%p) bcCurrent (0x%x) npeCauseMsg (%s) isMethodFlag (%d)"
```

Comment:
The signature of this tracepoint should be restored (and marked obsolete); a new one is required for a different signature.

Reply:
Since this is a preview feature, we decided it didn't matter - no one has seen this tracepoint in the wild.

Reply:
https://github.com/eclipse/openj9/pull/10912#discussion_r507946425
_Given that these tracepoints have never been used (preview feature), it might be best to simply change the format string in place._

Reply:
Fair enough. Glad to see that I wasn't the only one who considered this issue.